plugins/nf-validation/src/test/nextflow/validation/ValidateParametersTest.groovy

```diff
@@ -723,4 +723,84 @@ class ValidateParametersTest extends Dsl2Spec{
 '''
         !stdout
     }
+
+    def 'should fail because of wrong draft' () {
+        given:
+        def schema = Path.of('src/testResources/nextflow_schema_draft7.json').toAbsolutePath().toString()
+        def  SCRIPT_TEXT = """
+            params.monochrome_logs = true
+            include { validateParameters } from 'plugin/nf-validation'
+
+            validateParameters(parameters_schema: '$schema', monochrome_logs: params.monochrome_logs)
+        """
+
+        when:
+        dsl_eval(SCRIPT_TEXT)
+        def stdout = capture
+                .toString()
+                .readLines()
+                .findResults {it.contains('WARN nextflow.validation.SchemaValidator') || it.startsWith('* --') ? it : null }
+
+        then:
+        def error = thrown(SchemaValidationException)
+        !stdout
+    }
```

Comment:
Is there an explicit message we can check for? Maybe even just the presence of `draft` in the message or something?

Reply:
I haven't found a way to get the stderr in this case :/ Maybe I can find another workaround

Reply:
I think let's not worry too much for now. We can always improve the test in future.

Reply:
Fine by me :p